core/storage/storage_models_test.py

```diff
@@ -49,9 +49,11 @@ def _get_model_classes(self):
                     if 'Model' in all_base_classes:
                         yield clazz
 
-    # List of model classes that can't have Wipeout related class methods
+    # List of model classes that don't have Wipeout related class methods
     # defined because they're not used directly but only as a base classes for
-    # the other models.
+    # the other models. BaseCommitLogEntryModel is not included in this list
+    # because we implement has_reference_to_user_id inside it, the reason behind
```

Comment:
it, the reason behind this is that the --> it, since the

Reply:
Done.